code_examples/sdk_examples/src/core_features/did/00_generate_did_keys.ts

```diff
@@ -1,54 +1,33 @@
 import * as Kilt from '@kiltprotocol/sdk-js'
 
-import {
-  blake2AsU8a,
-  keyExtractPath,
-  keyFromPath,
-  mnemonicGenerate,
-  mnemonicToMiniSecret,
-  sr25519PairFromSeed
-} from '@polkadot/util-crypto'
+import { mnemonicGenerate } from '@polkadot/util-crypto'
 
-// Because there is no first-class support for this class of keys,
-// we need to use a workaround to generate a key we can use for encryption/decryption.
-function generateKeyAgreement(mnemonic: string): Kilt.KiltEncryptionKeypair {
-  const secretKeyPair = sr25519PairFromSeed(mnemonicToMiniSecret(mnemonic))
-  const { path } = keyExtractPath('//did//keyAgreement//0')
-  const { secretKey } = keyFromPath(secretKeyPair, path, 'sr25519')
-  return Kilt.Utils.Crypto.makeEncryptionKeypairFromSeed(blake2AsU8a(secretKey))
-}
+const signingKeyPairType = 'sr25519'
 
 export function generateKeypairs(mnemonic = mnemonicGenerate()): {
-  authentication: Kilt.KiltKeyringPair & {
-    type: 'sr25519'
-  }
+  authentication: Kilt.KiltKeyringPair
   keyAgreement: Kilt.KiltEncryptionKeypair
   assertionMethod: Kilt.KiltKeyringPair
   capabilityDelegation: Kilt.KiltKeyringPair
 } {
```

Comment:
This is nothing you changed here. But i feel like this is too complicated for no good reason.

Maybe we could simply pass a mnemonic or call mnemonicGenerate() inside the function?